packages/exchange/package.json

```diff
@@ -128,7 +128,9 @@
   },
   "dependencies": {
     "@yoroi/common": "1.5.1",
-    "zod": "^3.22.2"
+    "axios": "^1.5.0",
+    "react-query": "^3.39.3",
+    "zod": "^3.22.4"
```

Comment:
move anything that is not within the workspace as `peerDependencies` and `devDependencies`